packages/ckeditor5-utils/src/madewith.ts

```diff
@@ -36,19 +44,33 @@ export default function verify( token: string ): VerifiedKeyStatus {
 	const firstElement = splittedDecryptedData[ 0 ];
 	const secondElement = splittedDecryptedData[ 1 ];
 
+	if ( !secondElement ) {
+		return oldTokenCheck( token );
+	}
+
 	try {
-		// Must be a valid format.
-		atob( firstElement );
+		atob( secondElement );
```

Comment:
Why using second element at all? It's always going to be the last part.

Reply:
Because we're checking old vs new licence key - old licence keys after `atob` can contain `-` and it will split the string into array so it can look like a new licence key format. To verify that we need to check if second element is base64 coded string - if yes it's a new licence key format if not it's and old one.